oak/server/wasm_node.cc

```diff
@@ -227,6 +227,41 @@ void WasmNode::InitEnvironment(wabt::interp::Environment* env) {
       wabt::interp::FuncSignature(std::vector<wabt::Type>{wabtUsizeType, wabtUsizeType},
                                   std::vector<wabt::Type>{wabt::Type::I32}),
       this->OakRandomGet(env));
+
+  // Experimental.
```

Comment:
Could you add more details? What is WASI? Why is it experimental? Why these functions and not others? Why this particular implementation of the placeholder? etc. :)

Reply:
Done